src/ResourceManager/Search/Commands.Management.Search/SearchService/GetSearchServiceCommand.cs

```diff
@@ -0,0 +1,86 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Management.Search.SearchService
+{
+    [Cmdlet(VerbsCommon.Get, SearchServiceNounStr), OutputType(typeof(Models.PSSearchService))]
```

Comment:
Add defaultparametersetname (apply to all cmdlets with more than one parameter set).